tests/end_to_end/test_audit/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Test case configuration
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Get euid
+      shell: echo $EUID
+      register: euid
+    - debug:
+            var: euid.stdout
```

Comment:
Here you use indentation of 4 and below 2. Use the one used in the rest of the files.

Reply:
Done in eca0f3f